bank-application/src/main/java/com/cognologix/bankapplication/services/customers/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package com.cognologix.bankapplication.services.customers.impl;
+
+import com.cognologix.bankapplication.dto.UserAccountCreationResponse;
+import com.cognologix.bankapplication.exceptions.NullArgumentException;
+import com.cognologix.bankapplication.models.User;
+import com.cognologix.bankapplication.repositories.UserRepository;
+import com.cognologix.bankapplication.services.customers.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class UserServiceImpl implements UserService {
+
+    @Autowired
+    private UserRepository userRepository;
+
+    @Override
+    public UserAccountCreationResponse createUserAccount(User user) {
+        if (user.getCustomerName() == null || user.getAadharNumber() == null || user.getCustomerCity() == null
```

Comment:
you can have a validation method for this rather than checking this here, or may be annotation based validation